internal/graphql/resolver/meshsync.go

```diff
@@ -218,3 +232,20 @@ func (r *Resolver) connectToNats(ctx context.Context, provider models.Provider)
 	})
 	return model.StatusConnected, nil
 }
+
+type k8sConnectionTracker struct {
```

Comment:
Wow. We didn't have such a construct previously, huh? Should this be a struct or should this be an attribute of an existing connection object? Meshery has objects for connected entities, right? CRUD performed for k8s contexts -- is there not struct for those objects?

Do we have a central set of structs for connection tracking? If not, let's move toward such a model.

Reply:
@hexxdump @Zanda256 @MarioArriaga92 fyi

Reply:
This struct does not do anything special, other than reacting to delta of k8s context i.e. whenever k8s context is switched , it will react and reconnect to the broker with this new context.